umake/frameworks/ide.py

```diff
@@ -68,69 +67,66 @@ def __init__(self):
 
 class Eclipse(umake.frameworks.baseinstaller.BaseInstaller):
     """The Eclipse Foundation distribution."""
-    DOWNLOAD_URL_PAT = "https://www.eclipse.org/downloads/download.php?" \
-                       "file=/technology/epp/downloads/release/luna/R/" \
-                       "eclipse-standard-luna-R-linux-gtk{arch}.tar.gz{suf}" \
-                       "&r=1"
 
     def __init__(self, category):
         super().__init__(name="Eclipse",
-                         description=_("Pure Eclipse Luna (4.4)"),
+                         description=_("Eclipse Java"),
                          category=category, only_on_archs=['i386', 'amd64'],
-                         download_page=None,
+                         download_page='https://www.eclipse.org/downloads/',
+                         checksum_type=ChecksumType.sha512,
                          dir_to_decompress_in_tarball='eclipse',
                          desktop_filename='eclipse.desktop',
                          required_files_path=["eclipse"],
-                         packages_requirements=['openjdk-7-jdk'])
+                         packages_requirements=['openjdk-8-jdk'])
```

Comment:
Can this work with openjdk-7-jdk? 14.04 doesn't have openjdk-8-jdk and I would like to support it until the next LTS is released at least.